cogs/admin.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, bot):
                     color=0x419400
                 )
                 await ctx.send(embed=local_embed, delete_after=3)
-        except Exception as e:
+        #Variable 'e' is never used; except Exception as e:
```

Comment:
Rather than just comment it out - which will break the bot, it would be better here to just remove the `as e` so that the line reads `except Exception:`